core/com/lowagie/tools/BuildTutorial.java

```diff
@@ -171,22 +172,25 @@ public static void convert(File infile, File xslfile, File outfile) {
 		try {
 			// Create transformer factory
 			TransformerFactory factory = TransformerFactory.newInstance();
+			// XML Security is not strictly necessary in a user tool but is still added.
```

Comment:
we need FEATURE_SECURE_PROCESSING here, unfortunatelly it might not be compatible.

Reply:
The class BuildTutorial is not in any way a core iText class used in reading or writing PDF documents (though it is under /core/*). It is a helper tool to build a website for the iText project and is called during the ant build process (script ant/site.xml that we do not use).